components/base/base.go

```diff
@@ -49,13 +49,15 @@ type Base interface {
 
 	// Spin spins the robot by a given angle in degrees at a given speed.
 	// If a speed of 0 the base will stop.
+	// A positive angle is measured in degrees to the left.
+	// A positive speed turns to the left.
 	// This method blocks until completed or cancelled
 	Spin(ctx context.Context, angleDeg, degsPerSec float64, extra map[string]interface{}) error
 
 	SetPower(ctx context.Context, linear, angular r3.Vector, extra map[string]interface{}) error
 
-	// linear is in mmPerSec
-	// angular is in degsPerSec
+	// linear is in mmPerSec (positive Y moves forwards)
```

Comment:
```suggestion
	// linear is in mmPerSec (positive Y moves forwards for builtin rdk drivers)
```

And I'd add to SetPower too. I want to be clear that *we* have decided on this for our own drivers and that's what we expect for Motion commands, but you don't have to at all if you're implementing your own thing as a module, with your own motion logic.

Reply:
Wouldn't we want people to follow the same conventions? Sure they don't have to, but should we encourage it in some way?

Reply:
Sometimes, people prefer Z down or X forward for ground vehicles. There are a ton of applications with a ton of preferences. I'm unclear on how strongly we want to enforce this standard on someone who has enough understanding to flip around the frame system.

Reply:
I do not think this language strongly encourages anything anyway. It's more informative than prescriptive in both cases.

Reply:
okay--do we need to put the "for built-in RDK drivers" on all of these descriptions then though?

Reply:
let's hold off on the other methods (Spin, MoveStraight) until we make the agileX a module that does things slightly differently as it's an experimental package for a base that implements Ackerman, omni, and differential steering and does different things with spin because you can only send the base velocity commands and we time it out for those two methods. When we move it out we're good to go on updating the docs.

Reply:
Ooh so the changes I was trying to make to the Spin comments in this PR shouldn't be made because it's not even the same across our bases?

Reply:
yeah. ultimately - we have two bases in rdk, and I expect that to stay because most likely everyone will implement their own base in their module, and they really vary based on the drivetrain. Particularly in what spin can do when you can steer.

Reply:
```suggestion
	// linear is in mmPerSec (positive Y moves forwards for built-in RDK drivers)
```